tileserver/__init__.py

```diff
@@ -135,6 +135,29 @@ def decode_json_tile_for_layers(tile_data, layer_data):
     return feature_layers
 
 
+def select_layers_from_tile(tile_data, layer_data, coord, format):
```

Comment:
Yup, that's a good point. Renamed to `reformat_selected_layers` in b2a933bdc11448f197eb24a5cb2821ada90ea742 and fixed the parameter in 8091b7d28c3eb518b5f52cdc60c426467087d5be.